src/screens/SettingsSheet/components/DevSection.tsx

```diff
@@ -188,6 +188,7 @@ const DevSection = () => {
       if (shouldWipeKeychain) {
         await wipeKeychain();
         await clearMMKVStorage();
```

Comment:
Should be good to follow up with this, but I think some of the MMKV instances being reset in `clearMMKVStorage` aren't necessary to reset when the intention is only to reset wallets/the keychain. May not be needed at all, or at least likely only specific `STORAGE_IDS` need clearing, e.g. `FIRST_APP_LAUNCH`:

https://github.com/rainbow-me/rainbow/blob/d31b1185e9e5c51a29a59c90fd5982f63bcccef9/src/model/mmkv.ts#L3-L17

Also, assuming `wipeKeychain` is important to resetting wallets state fully, would be good to move that into `clearWalletState` (in the `resetKeychain` path) so that `clearWalletState` can cleanly be used on its own to fully reset, along with handling any necessary MMKV storage resetting.